### PR TITLE
Feature/ui 25 panel and basic table

### PR DESCRIPTION
**Pull request by lmiller1990**
This will likely conflict with  @julianjuko's smart list improvements since I am sharing some of the types.

Panel

image

Basic List
image

### CODE REVIEW

File: src/style/colors.ts [hunk 1]
```diff
@@ -20,3 +20,34 @@ export const NAVY = 'rgb(14, 78, 149)'
 export const DROPDOWN_BLUE = 'rgb(197, 218, 246)'
 export const LIGHTEST_GREY = 'rgb(252, 252, 252)'
 export const CIRCLE_BUTTON_BLUE = 'rgb(43, 82, 136)'
+export const SKYBLUE = 'rgb(241, 246, 253)'
+
+interface IColor {
+  name: string
+  color: string
+}
+export const colors: IColor[] = [
```
Comment (lmiller1990) [inline src/style/colors.ts:29]: useful for stories

File: src/Panel/Panel.tsx [hunk 2]
```diff
@@ -20,18 +20,24 @@ const Panel: React.FC<IProps> = ({ title, info, bodyClassName, children }) => {
...
-    return (
-      <PanelBody bodyClassName={bodyClassName}>
-        {children}
-      </PanelBody>
-    )
-  }
+  const body: JSX.Element = (
+    <PanelBody bodyClassName={bodyClassName}>
+      {
+        sections?.map(x =>
```
Comment (MichaelDorkhom) [inline src/Panel/Panel.tsx:26]: Why do you need `?` here?

File: src/Panel/Panel.tsx [hunk 3]
```diff
@@ -20,18 +20,24 @@ const Panel: React.FC<IProps> = ({ title, info, bodyClassName, children }) => {
...
-      </PanelBody>
-    )
-  }
+  const body: JSX.Element = (
+    <PanelBody bodyClassName={bodyClassName}>
+      {
+        sections?.map(x => 
+          <div className='d-flex'>
+            <div className='panel-section-side'></div>
+            <div className='panel-section' key={x.id}>{x.element}</div>
```
Comment (MichaelDorkhom) [inline src/Panel/Panel.tsx:29]: Can you change the key to something like `panel-section-${x.id}`

File: src/Panel/Panel.tsx (hunk 2 above)
Comment (lmiller1990) [reply src/Panel/Panel.tsx]: removed

**Pull request merged**